metaci/release/utils.py

```diff
@@ -159,11 +203,12 @@ def send_start_webhook(project_config, release, role, config_item):
         raise Exception("\n".join(err["message"] for err in result["errors"]))
 
 
-def send_stop_webhook(project_config, release, role, config_item):
+def send_stop_webhook(release, role, config_item):
     if (
         release is None
         or not settings.METACI_RELEASE_WEBHOOK_URL
         or not settings.METACI_START_STOP_WEBHOOK
+        or not settings.GUS_BUS_OWNER_ID
```

Comment:
Don't need this check; the owner was set when the implementation step was created.